packages/core/src/DirectoryLoader.ts

```diff
@@ -333,6 +344,15 @@ export class PackageDirectoryLoader extends DirectoryLoader {
 			}
 		}
 
+		if (Array.isArray(credentials)) {
```

Comment:
since we swapped the order here, we should check that we aren't breaking any of the community nodes.